src/style.css

```
@@ -539,16 +485,20 @@ h2:hover a .anchor-hint {
 }
 
 .language-select {
-    padding-top: 0.15rem;
-    padding-right: 0.4rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 0.75rem;
 }
 
 .language-select span:not(:last-child):after {
     content: " | ";
+    padding-left: 0.75rem;
 }
 
-.language-select span.current {
+.language-select span.current a{
     font-weight: bold;
+    text-decoration-style: double;
 }
 
```

Small suggestion to make it fit a bit more tightly on smaller viewports:

```suggestion
@media (max-width: 1024px) {
    .utility-bar {
        padding-top: 1rem;
        padding-right: 1rem;
    }
}
```